### PR TITLE
Test against ort-nightly when packaging Azure ops

### PR DESCRIPTION
**Pull request by RandySheriffH**
Test against ort-nightly when packaging Azure ops.

### CODE REVIEW

File: tools/test_cibuildwheel.bat [hunk 1]
```diff
@@ -1,7 +1,8 @@
 @echo off
 if "%OCOS_ENABLE_AZURE%"=="1" (
     pushd %1\test
-    python -m pip install onnxruntime
+    python -m pip install coloredlogs flatbuffers numpy packaging protobuf sympy
+    python -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ ort-nightly==1.16.0.dev20230806001
```
Comment (wenbingl) [inline tools/test_cibuildwheel.bat:5]: I think the nightly version will only keep for some time and then was deleted. Is this what you want?

File: tools/test_cibuildwheel.bat [hunk 2]
```diff
@@ -1,7 +1,8 @@
 @echo off
 if "%OCOS_ENABLE_AZURE%"=="1" (
     pushd %1\test
-    python -m pip install onnxruntime
+    python -m pip install coloredlogs flatbuffers numpy packaging protobuf sympy
```
Comment (wenbingl) [inline tools/test_cibuildwheel.bat:4]: Are these ort's dependencies?

File: tools/test_cibuildwheel.bat (hunk 1 above)
Comment (RandySheriffH) [reply tools/test_cibuildwheel.bat:5]: Shortly, we will update the version to the RC.
For now, let's just keep the latest one at hand.

File: tools/test_cibuildwheel.bat (hunk 2 above)
Comment (RandySheriffH) [reply tools/test_cibuildwheel.bat:4]: that's right.

**Review by wenbingl — APPROVED** [review]

**Pull request merged**